robosat/tools/merge.py

```diff
@@ -65,6 +65,9 @@ def unbuffered(shape):
         merged = unbuffered(union(embiggened))
 
         if merged.is_valid:
+            # Orient exterior ring of the polygon in counter-clockwise direction.
+            merged = shapely.geometry.polygon.orient(merged, sign=1.0)
```

Comment:
Could you add a comment why we need this. Is this coming from the union call above?

Otherwise looks good, you can merge afterwards.

Reply:
Posted details here: https://github.com/mapbox/robosat/pull/64#issuecomment-410165296